examples/inplace/file.txt

```diff
@@ -1 +1 @@
-The roof is on fire!
```

Comment:
This is weird, was the example wrong the entire time?

Reply:
Nothing wrong. It was modified because I have run `gulp` in the example and restored this file manually, instead of discarding changes before committing. It is exactly the same as the original file.

Reply:
It was modified because I have run `gulp` in the example. At present, I have restored the example.